### ISSUE TITLE
Mentions being rendered twice

### ISSUE THREAD

**Issue by luisrudge** [jpuri/react-draft-wysiwyg#216]
This is happening after I converted to HTML, so I'm not sure it's an issue with this project or with https://github.com/jpuri/draftjs-to-html.

![screen shot 2017-02-13 at 23 02 42](https://cloud.githubusercontent.com/assets/941075/22910356/8ea78808-f240-11e6-8ca0-7d3d35b522f1.png)

```js
{
    "entityMap": {
        "0": {
            "type": "MENTION",
            "mutability": "IMMUTABLE",
            "data": {
                "text": "#ábaco",
                "value": "ábaco",
                "url": "/t?id=A463F565-59C3-49C1-B47C-3EBBB7FBC0E3"
            }
        }
    },
    "blocks": [{
        "key": "8bvet",
        "text": "test #ábaco ",
        "type": "unstyled",
        "depth": 0,
        "inlineStyleRanges": [],
        "entityRanges": [{
            "offset": 5,
            "length": 6,
            "key": 0
        }],
        "data": {}
    }]
}
```

This is being converted to:

```html
<p>test <a href="/t?id=A463F565-59C3-49C1-B47C-3EBBB7FBC0E3" class="wysiwyg-mention" data-mention data-value="ábaco">#ábaco</a><a href="#ábaco" class="wysiwyg-hashtag">#ábaco</a>&nbsp;</p>

```

As you can see, there's two links there.

**Comment by jpuri** [comment]
Hey @luisrudge, I see this is due to a recent change in api for draftjsToHtml. Hmmm I am sorry for the breaking change but I have added a param to `draftToHtml(contentState, hashtagConfig, directional, customEntityTransform)` check : https://github.com/jpuri/draftjs-to-html.

If second param is not null hashtags are generated since `#` is default starting character for hashtags and you are using it for mentions its creating trouble. Its edge case as you are using `#` for mentions. This will be fixed if you make second param `undefined`.

Changing api is wrong on my part, I will more careful about potentially breaking changes hence forth.

**Comment by luisrudge** [comment]
np, @jpuri thanks for your help!

I'm using it like this and still having the same issue: `term.content = draftToHtml(contentState, undefined);`

**Comment by jpuri** [comment]
Checking that.

**Comment by jpuri** [comment]
Here is fix `draftjs-to-html@0.6.3`

**Comment by luisrudge** [comment]
That's perfect. Thank you! 🎉
